Assets/Scripts/Menu Scripts/RadiationButtons.cs

```diff
@@ -29,44 +28,57 @@ void OnClickEvent()
         if (surface_pointcloud)
```

Comment:
im confused, what exactly does this do because all the if statements have the same block of code except for the debug statement.

Reply:
Just a placeholder till Jasmine is done implementing the UI